### DISCUSSION TITLE
Inquiry about Installing DAT Linux Distribution ISO

### DISCUSSION THREAD

**Discussion by ghost** [dat-linux/community#54] [FAQ]
I have little experience with Linux and was wondering if you could provide some assistance on how to install the ISO image. Specifically, I'm unsure whether the ISO needs to be booted from a flash drive or if there's an alternative method for installation.

As a Windows 10 user with limited Linux experience, any detailed instructions or guidance you can offer would be greatly appreciated. I'm excited to dive into the world of DAT Linux and your expertise would be invaluable in helping me get started.

**Comment by dat-linux** [comment]
Hi, create a bootable flash drive and boot into it on restart or use virtualbox

there are many resources online for how to do either of these

**Reply by ghost** [reply]
Yes i have been able to install it and it works super fine

however, i am still unable to start a quarto project in R studio. As far as i can notice, no option for quarto project or file is showing. I tried many times but i could not figure out how to. A bit odd, because it is a main part of my professional duties to work with quarto. I acknowledge to be a bit lost (coming from windows)

is there any help you could provide ?
